### PR TITLE
initial component sheet

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 
the very beginnings of getting the component sheet up and running

### CODE REVIEW

File: react-native/ios/Keybase.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -671,7 +671,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "../node_modules/react-native/packager/react-native-xcode.sh";
+			shellScript = "export NODE_BINARY=node\n../node_modules/react-native/packager/react-native-xcode.sh";
```
Comment (chrisnojima) [inline react-native/ios/Keybase.xcodeproj/project.pbxproj]: update from react-native upgrade

File: react-native/.watchmanconfig [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (chrisnojima) [inline react-native/.watchmanconfig]: from react-native upgrade

File: shared/common-adapters/box.desktop.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React, {Component} from 'react'
+
+export default class Box extends Component {
```
Comment (chrisnojima) [inline shared/common-adapters/box.desktop.js]: This is a wrapper around div/View. let's discuss if this makes sense

File: shared/more/style-sheet.render.native.js [hunk 4]
```diff
@@ -1,12 +1,240 @@
...
+        <Container title='Icons'>
+          <Text>TODO</Text>
+        </Container>
+        <Container title='Inputs'>
+          <Text>TODO</Text>
+        </Container>
+      </ScrollView>
+    )
+
+    // return (
```
Comment (MarcoPolo) [inline shared/more/style-sheet.render.native.js]: I'd prefer not commiting this since it's just a commented out copy of style-sheet.desktop, right?

**Comment by MarcoPolo** [comment]
:+1: % commented code

File: shared/more/style-sheet.render.native.js (hunk 4 above)
Comment (chrisnojima) [inline shared/more/style-sheet.render.native.js]: its all going to get uncommented basically

**Comment by MarcoPolo** [comment]
feel free to merge the commented code isn't a big deal

**Pull request merged**